dev-db/redis/metadata.xml

```diff
@@ -1,10 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-	<maintainer type="person">
+	<maintainer type="person" proxied="proxy">
```

Comment:
FWIW, and this is confusing, this actually means I don't maintain it in my own right IIRC: https://gitweb.gentoo.org/data/glep.git/commit/?id=824f0fcd9a466ca69e9705bd2f380ccac768151a

So, I'll drop that part, and also put you first, just because I've been feeling guilty about the attention (or lack of) I've been giving to Redis, and you've been doing a good job. Happy to change it though.